packages/manager/core/generator/app/templates/src/components/Error/error.scss

```diff
@@ -0,0 +1,16 @@
+.manager-error-page {
+  margin-left: auto;
+  margin-right: auto;
+  width: 600px;
+  display: grid;
+  height: 100%;
+  overflow: auto;
+
+  div {
+    padding: 15px;
```

Comment:
**nitpick:** To maintain the uniformity across the Manager, use bootstrap utility classes.

Reply:
are we sure we want to continue with this library in react ? ods will publish grid and layout